source/includes/kubernetes/_k8_releases.md

```diff
@@ -134,162 +157,167 @@ The information is not totally returned in the list. We filter out the manifest
+| `chart.metadata.description` <br/>_string_ | The description of the chart.                                                                                                                                                           |
+| `chart.metadata.keywords` <br/>_list_      | The list of keywords linked to the chart.                                                                                                                                               |
+| `chart.metadata.maintainers` <br/>_list_   | The list of the maintainer contact information.                                                                                                                                         |
+| `chart.metadata.icon` <br/>_string_        | The icon URL for the chart.                                                                                                                                                             |
+| `chart.metadata.appVersion` <br/>_string_  | The version of the application.                                                                                                                                                         |
+| `chart.metadata.deprecate` <br/>_bool_     | If the chart is deprecated or not of the application.                                                                                                                                   |
+| `chart.templates` <br/> _list_             | The list of templates contained inside the chart.                                                                                                                                       |
+| `chart.templates.name` <br/> _string_      | The path name of the template inside the chart.                                                                                                                                         |
+| `chart.templates.data` <br/> _string_      | The contents of the template. This is a base64 encode string.                                                                                                                           |
+| `chart.files` <br/> _list_                 | The list of files contained inside the chart. These are not YAML files unlike the templates.                                                                                            |
+| `chart.files.name` <br/> _string_          | The path name of the file inside the chart.                                                                                                                                             |
+| `chart.files.data` <br/> _string_          | The contents of the file. This is a base64 encode string.                                                                                                                               |
```

Comment:
```suggestion
| `chart.files.data` <br/> _string_          | The contents of the file. This is a base64 encoded string.                                                                                                                               |
```

Reply:
find and replace references

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/261/commits/498b4777ba7f890f506f25c754059b8484c23de2#diff-392b4cde4e74dbeef62e28257656cdebf7338dbf535e526c65e374706ebc0330R313